src/ray/object_manager/plasma/store.h

```diff
@@ -210,8 +210,14 @@ class PlasmaStore {
   /// Process queued requests to create an object.
   void ProcessCreateRequests();
 
+  /// Get the available memory for new objects to be created. This includes
+  /// memory that is currently being used for created but unsealed objects.
   void GetAvailableMemory(std::function<void(size_t)> callback) const {
-    int64_t num_bytes_in_use = static_cast<int64_t>(num_bytes_in_use_);
+    RAY_CHECK(num_bytes_in_use_ >= num_bytes_unsealed_);
+    // We include unsealed objects because these may have been created by the
```

Comment:
Isn't it exclude?

Reply:
Also, how big are unsealed objects are usually? (Do they contain the data in it? Or is data updated after it is sealed)?

Reply:
Ah yes, I'll update the comment.

Unsealed objects are as large as the created object. The plasma store allocates a buffer of the requested data size, and the client "seals" it once the data has been copied into the buffer.